backend/FwLite/MiniLcm/Validators/CanonicalGuidsSemanticDomain.cs

```diff
@@ -0,0 +1,1803 @@
+﻿using FluentValidation;
+
+namespace MiniLcm.Validators;
+
+public static class CanonicalGuidsSemanticDomain
+{
+    // GUID list taken from src/SIL.LCModel/Templates/SemDom.xml in liblcm
+    // TODO: Consider loading SemDom.xml into app as a resource and add singleton providing access to it, then look up GUIDs there rather than using this hardcoded list
+    public static HashSet<Guid> CanonicalSemDomGuids = [
+        new Guid("63403699-07C1-43F3-A47C-069D6E4316E5"),
```

Comment:
I don't want to spend a bunch of time on it if it's not easy, but looking at this code, at runtime it's going to have to parse all 1,000 strings. An alternative would be to use the constructor which takes a `ReadOnlySpan<byte>`. This would require creating the guids via parsing then calling `guid.ToByteArray()` and converting that into source like this:
```C#
new Guid([128, 117, 208, 48, 82, 80, 145, 77, 188, 36, 70, 155, 139, 45, 125, 249]), //30d07580-5052-4d91-bc24-469b8b2d7df9
```
again if that turns out to be too hard don't worry about it, we can always improve it later

Reply:
Will do later. It's probably easy to automate this with a quick script, so I'll try that.

Reply:
Opened #1361 to track this remaining bit of work; will resolve this conversation and merge the PR since it's been otherwise approved.